ui/ducks/swaps/swaps.js

```diff
@@ -682,7 +715,10 @@ export const signAndSendTransactions = (history, metaMetricsEvent) => {
       .plus(usedQuote.approvalNeeded?.gas || '0x0', 16)
       .toString(16);
     const gasEstimateTotalInUSD = getValueFromWeiHex({
-      value: calcGasTotal(totalGasLimitEstimate, usedGasPrice),
+      value: calcGasTotal(
+        totalGasLimitEstimate,
+        EIP1559Network ? maxFeePerGas : usedGasPrice,
```

Comment:
I think this should be `baseFeePerGas + maxPriorityFeePerGas`

Reply:
`usedGasPrice` is use to calculate the "estimate", which is what `baseFeePerGas + maxPriorityFeePerGas` would be used for